projects/standard-rulesets/src/breaking-changes/__tests__/parameter-enum-breaking-changes.test.ts

```diff
@@ -108,6 +108,117 @@ describe('breaking changes ruleset - parameter enum change', () => {
       );
       expect(results.length > 0).toBe(true);
 
+      expect(results).toMatchSnapshot();
+      expect(results.some((result) => !result.passed)).toBe(true);
+    }
+  );
+  test.each(['query', 'cookie', 'path', 'header'])(
+    'enum converted to const %p parameter',
+    async (location: string) => {
+      const beforeJson: OpenAPIV3.Document = {
+        ...TestHelpers.createEmptySpec(),
+        paths: {
+          '/api/users': {
+            get: {
+              parameters: [
+                {
+                  name: 'test',
+                  in: location,
+                  schema: {
+                    type: 'string',
+                    const: 'a',
+                    // Casting because const does not exist in OpenAPIV3.NonArraySchemaObject
+                  } as OpenAPIV3.NonArraySchemaObject,
```

Comment:
I don't love this, but it seems to be an external-to-optic library. I tried to at least scope the casts to the minimum size I could (vs casting the whole `OpenAPIV3.Document`

Reply:
Yeah it looks like we have a few typing issues to think through before we can fix this. Since Optic currently won't even read the `const` schemas we'll get that work up and ping you when you can use the correct typing. Probably going to take a day or two -- we're launching a few other features today first.